modules/ROOT/pages/fd-connection-create.adoc

```diff
@@ -0,0 +1,59 @@
+<4> In the procedure explained in the next section, you click the *Add connection* button to create a connection.
+<5> You return to the configuration dialog for the connector, so that you can specify values for fields in the *Advanced* section, if you need to do so.
+
+A part of the configuration dialog that presents options about connections looks like this:
+
+.Part of the *Database Configuration* dialog presents a list of connections to databases and an option for creating a connection to a database.
+image::fd-connection-add.png[The locations of the list and the option]
+
+<1> Each existing connection shows roughly when the connection was created and the user ID of the person who created it.
+<2> The *Add connection* option lets you create a connection.
+
+You can also create a connection if you are editing a configuration or if you are creating an additional configuration for a connector.
+
+.The locations from which you can open the configuration dialog for a connector and add a connection.
+image::fd-edit-switch-configs.png[Where to go to add a connection]
+
+<1> When there is at least one configuration for a connector, the top of the connector card looks like this. You can edit the configuration or switch to a different configuration. If you edit the configuration, the configuration dialog is opened and you can create a connection.
+<2> If you select the option to switch to a different configuration, the top of the card changes to look like this. You can click *Add* to open the configuration dialog for the connector and create a connection.
+
+== To Add a Connection to a Configuration of a Connector
+
+. Click *Add connection* in the configuration dialog for a connector that you have added to a flow.
+. Give the connection a descriptive name.
+. If you want the connection to be available to yourself and others in your business group for use in other projects, select the *Share this connection with my business group* checkbox.
++
+.The *Share this connection with my business group* checkbox is located after the *Connector Name* field.
+image::fd-connection-checkbox.png[The location of the *Share this connection with my business group* checkbox]
+. Select the type of connection that you are creating. For help with the fields, see the documentation for the connector that you are adding to your Mule-app project.
+. Click *Save*.
+
+== Result
+
+Flow Designer returns you to the configuration dialog for the connector. The connection that you added is selected as the connection for the configuration to use. Click *Save* to return to the canvas.
```

Comment:
I would move all this up into the task. Only one sentence here is the result, and we generally assume that the result is as intended (Add).